add Linux 5.15.131 packages

cfm opened the pull request:
## Status

Ready for review

## Description of changes

Originally [intended][1] just to rebuild #202 with
freedomofpress/kernel-builder#38, this also moved us from v5.15.30 to v5.15.131.

Towards:
- freedomofpress/securedrop#6938
- freedomofpress/securedrop-workstation#916

[1]: https://github.com/freedomofpress/securedrop/issues/6938#issuecomment-1709126141

## Checklist
- [ ] CI passes.
- [ ] Manual verification confirms that freedomofpress/kernel-builder#38 took effect, e.g.:

    ```sh-session
    user@sd-dev:~/securedrop-apt-test$ debdiff core/focal/securedrop-grsec_5.15.130-1-grsec-securedrop-1_amd64.deb core/focal/securedrop-grsec_5.15.131-1-grsec-securedrop-1_amd64.deb 
    File lists identical (after any substitutions)

    Control files: lines which differ (wdiff format)
    ------------------------------------------------
    Depends: [-linux-image-5.15.130-1-grsec-securedrop,-] {+linux-image-5.15.131-1-grsec-securedrop,+} intel-microcode, amd64-microcode, [-paxctld, linux-image-5.15.89-grsec-securedrop-] {+paxctld+}
    Version: [-5.15.130-1-grsec-securedrop-1-] {+5.15.131-1-grsec-securedrop-1+}
    ```

- [ ] Build logs have been committed to [build-logs](https://github.com/freedomofpress/build-logs): freedomofpress/build-logs@c2137c0afb3999193875e612a2ff9ee621d875aa

rocodes reviewed (approved):
- [x] CI passing / hashes match build logs
- [x] Build logs committed 

:shipit:

cfm commented:
NB. 55e2cb7c5482d8ec6e7f2a151668a0967d18ccb2 contains a typo and should read:

```patch
 Originally intended just to rebuild https://github.com/freedomofpress/securedrop-apt-test/pull/202 with
-https://github.com/freedomofpress/kernel-builder/pull/38[1], this also moved us from v5.15.30 to
+https://github.com/freedomofpress/kernel-builder/pull/38[1], this also moved us from v5.15.130 to
 v5.15.131.

 Towards:
 - https://github.com/freedomofpress/securedrop/issues/6938
 - https://github.com/freedomofpress/securedrop-workstation/issues/916

 [1]: https://github.com/freedomofpress/securedrop/issues/6938#issuecomment-1709126141
```
